public/index.html

```diff
@@ -9,6 +9,7 @@
       name="description"
       content="Web site created using create-react-app"
     />
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
```

Comment:
Install fontawesome via npm instead, so that the version is more easily controlled

Reply:
Wait, I see he installed it via NPM, shouldn't this be removed now? @luism-s @MarianBadea

Reply:
Yes, I removed it now, I didn't know and I import the fontawesome library in to App.js to use the icons